<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CSVImporterDialog</class>
 <widget class="QDialog" name="CSVImporterDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>913</width>
    <height>660</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CSV Importer</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Import selection</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QTreeWidget" name="m_QTreeWidget_ImportSelection">
         <property name="selectionMode">
          <enum>QAbstractItemView::MultiSelection</enum>
         </property>
         <attribute name="headerShowSortIndicator" stdset="0">
          <bool>true</bool>
         </attribute>
         <column>
          <property name="text">
           <string>CSV header</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Database column name</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Import data</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout" stretch="60,1">
       <item>
        <widget class="QTableWidget" name="m_QTableWidget_ImportData">
         <attribute name="horizontalHeaderVisible">
          <bool>true</bool>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="m_QTableWidget_ImportData_Totals"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="m_QDialogButtonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>m_QDialogButtonBox</sender>
   <signal>rejected()</signal>
   <receiver>CSVImporterDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
